perun-core/src/main/java/cz/metacentrum/perun/core/impl/UsersManagerImpl.java

```diff
@@ -183,6 +183,7 @@ public User getUserByUserExtSource(PerunSession sess, UserExtSource userExtSourc
 					" from users, user_ext_sources " +
 					"where users.id=user_ext_sources.user_id and user_ext_sources.login_ext=? and user_ext_sources.ext_sources_id=? ", USER_MAPPER, userExtSource.getLogin(), userExtSource.getExtSource().getId());
 		} catch (EmptyResultDataAccessException ex) {
+			log.debug("GET_USER_BY_USER_EXT_SOURCE");
```

Comment:
Again, this change probably shouldn't be a part of this pull-request, since it seems to be related to ExtSources and purpose is debugging it.